context/opencensus-context/README.rst

```diff
@@ -5,3 +5,71 @@ OpenCensus Runtime Context
 
 .. |pypi| image:: https://badge.fury.io/py/opencensus-context.svg
    :target: https://pypi.org/project/opencensus-context/
+
+The **OpenCensus Runtime Context** provides in-process context propagation.
+By default, thread local storage is used for Python 2.7, 3.4 and 3.5;
+contextvars is used for Python >= 3.6, which provides asyncio support.
+
+Installation
+------------
+
+This library is installed by default with `opencensus`, there is no need
+to install it explicitly.
+
+Usage
+-----
+
+In most cases, the context would just work for you automatically. This is
+achieved by leveraging `threading.local()` and `contextvars`.
```

Comment:
How about this?

"In most cases context propagation happens automatically within a process, following the control flow of threads and asynchronous coroutines. The runtime context is a dictionary stored in a [context variable](https://docs.python.org/3/library/contextvars.html) when available, and in [thread local storage](https://docs.python.org/2/library/threading.html#threading.local) otherwise."

Reply:
Taken :)